holidays/financial/national_stock_exchange_of_india.py

```diff
@@ -67,8 +94,15 @@ def _populate_public_holidays(self):
         # Dr. Baba Saheb Ambedkar Jayanti.
```

Comment:
```suggestion
        # Dr. B. R. Ambedkar Jayanti.
```
Msg comment should match en_US translation.